packages/mint-components/src/components/sqm-referral-table/cells/sqm-referral-table-rewards-cell.tsx

```diff
@@ -106,21 +116,29 @@ export class ReferralTableRewardsCell {
             {/* Pending for W9 Tax reasons cases here */}
             <div class={sheet.classes.BadgeContainer}>
               {state === "PENDING" && reward.dateScheduledFor ? (
-                <sl-badge type={slBadgeType} pill>{`${badgeText} ${
+                <sl-badge
+                  class={sheet.classes.StatusBadge}
+                  type={slBadgeType}
+                  pill
+                >{`${badgeText} ${
                   getTimeDiff(reward.dateScheduledFor) === "tomorrow" ||
                   getTimeDiff(reward.dateScheduledFor) === "today" ||
-                  getTimeDiff(reward.dateExpires) === "next month" ||
-                  getTimeDiff(reward.dateExpires) === "next year"
+                  getTimeDiff(reward.dateScheduledFor) === "next month" ||
```

Comment:
This should just be `Intl` for translation

Reply:
We can probably make this a more re-usable utility function, but intl's version is a bit underwhelming, as you still need to calculate which of hours/days/month etc it should use (see https://stackoverflow.com/questions/67373795/how-to-use-intl-formatrelativetime-in-the-correct-way)